src/rpc/common/Validators.cpp

```diff
@@ -200,16 +200,10 @@ CustomValidator CustomValidators::SubscribeStreamValidator =
             "ledger", "transactions", "transactions_proposed", "book_changes", "manifests", "validations"
         };
 
-        static std::unordered_set<std::string> const reportingNotSupportStreams = {
```

Comment:
we can create a new ClioError "NotSupported". Users may feel strange that such stream is valid according to document.